atlassian_jwt_auth/contrib/flask_app/decorators.py

```diff
@@ -0,0 +1,30 @@
+from functools import wraps
+
+from flask import Response, current_app, request
+
+import atlassian_jwt_auth
+from atlassian_jwt_auth.contrib.flask_app.utils import parse_jwt
+
+
+def requires_asap(f):
+    """
+    Wrapper for Flask endpoints to make them require authentication to access.
+    """
+
+    @wraps(f)
+    def decorated(*args, **kwargs):
+        verifier = _get_verifier()
+        auth = request.headers.get('Authorization')
```

Comment:
No, that's one of the differences between `request.headers['Authorization']` and `request.headers.get('Authorization')`. The latter will return `None`. The former will raise a `KeyError`